[2024-08-12T16:37:18.161+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-12T16:37:18.169+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-12T16:37:18.170+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-12T16:37:18.170+0000] {taskinstance.py:1363} INFO - Starting attempt 6 of 6
[2024-08-12T16:37:18.171+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-12T16:37:18.182+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-08-11 00:00:00+00:00
[2024-08-12T16:37:18.188+0000] {standard_task_runner.py:55} INFO - Started process 19715 to run task
[2024-08-12T16:37:18.192+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-08-11T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmps3pktol3']
[2024-08-12T16:37:18.195+0000] {standard_task_runner.py:83} INFO - Job 42: Subtask create_table
[2024-08-12T16:37:18.268+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [running]> on host 7b166f9b76f0
[2024-08-12T16:37:18.337+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-11T00:00:00+00:00
[2024-08-12T16:37:18.352+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-08-12T16:37:18.360+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
            firstname  TEXT NOT NULL,
            lastname TEXT NOT NULL,
            country TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
            );
        , parameters: None
[2024-08-12T16:37:18.362+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-08-12T16:37:18.373+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240811T000000, start_date=20240812T163718, end_date=20240812T163718
[2024-08-12T16:37:18.407+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-08-12T16:37:18.437+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
